samples/githubDispatch/README.md

```diff
@@ -0,0 +1,16 @@
+# /fusebot github-dispatch <user/repo> <workflow_id> <branch/tag name>
+
+This command triggers a github action with workflow_trigger action.
+
+1. Type `/fusebot edit github-dispatch`.
+2. Click the _edit_ link.
+3. Copy and paste the [command.js](command.js) file content into the command.js file in the web editor.
+4. Add `"octokit": "1.0.5"` into package.json as a dependency.
+5. Set a personal access token in configuration with the key name of PAT.
```

Comment:
Take into account people reading this may have little to no familiarity with Fusebit. Most concepts are new to them. You have to speak to them like to a 3-year-old. It is in our best interest to lay it down for them so that they feel empowered and taken care of without feeling stupid or inferior. What is "personal access token"? What is "configuration"? What does "PAT" stand for?

Consider an alternative: 

> In the *Configuration* section of the editor, set the `PAT={github-personal-access-token}` property. You can obtain your GitHub personal access token from {...whatever most useful URL or instructions to get it you can provide...}

Feel free to improvise.

Reply:
Done